### PR TITLE
additional subdomains for freecycle.org

### PR DESCRIPTION
**Pull request by KommX**

### CODE REVIEW

File: src/chrome/content/rules/Freecycle.xml [hunk 1]
```diff
@@ -4,10 +4,13 @@
...
+	<target host="my.freecycle.org"/>
+	<target host="www.freecycle.org"/>
+	<target host="freecycle.org"/>
+	<target host="groups.freecycle.org"/>
 
 	<securecookie host="^(?:web)?mail\.freecycle\.org$" name=".*"/>
 
-	<rule from="^http:"
-		to="https:" />
+	<rule from="^http:"	to="https:" />
```
Comment (reedy) [inline src/chrome/content/rules/Freecycle.xml]: Any reason to change this to a single line? Generally we don't "care" too much for stylistic changes like this

Comment (KommX) [inline src/chrome/content/rules/Freecycle.xml]: No, no particular reason. I just saw that 'make-trivial-rule' does it like that. The main part are the additional subdomains here.

File: src/chrome/content/rules/Freecycle.xml [hunk 2]
```diff
@@ -4,10 +4,13 @@
 	<target host="butler.freecycle.org"/>
 	<target host="mail.freecycle.org"/>
 	<target host="webmail.freecycle.org"/>
+	<target host="my.freecycle.org"/>
+	<target host="www.freecycle.org"/>
+	<target host="freecycle.org"/>
```
Comment (reedy) [inline src/chrome/content/rules/Freecycle.xml]: https://freecycle.org/ gives a 403

**Comment by KommX** [comment]
Yes, you're right. I now removed the target host 'freecycle.org'.

**Pull request merged**